server/src/metadata/metadata.resolver.ts

```diff
@@ -15,11 +15,6 @@ import { CreateCustomFieldInput } from './args/create-custom-field.input';
 export class MetadataResolver {
   constructor(private readonly metadataService: MetadataService) {}
 
-  @Query(() => String)
```

Comment:
Not needed anymore, you can't have a schema without a query but now we have plenty.